### DISCUSSION TITLE
lists.css

### DISCUSSION THREAD

**Discussion by argyleink** [argyleink/open-props#73] [Ideas]
common list-styles as props

**Comment by felix-bohlin** [comment]
Hey there @argyleink! Is this regarding list-styles like resetting margin, padding and different list-styles or something more? 

Otherwise this could be a part of a "prose"-type of pack for all type of rich text. Not sure Open Props wants to go that route though.

Happy holidays, new year and all that 🥳

**Reply by argyleink** [reply]
it's about creating well named values for the [`list-style`](https://developer.mozilla.org/en-US/docs/Web/CSS/list-style) shorthand!
